tonic/dart_state.h

```diff
@@ -21,7 +24,17 @@ class DartState : public tonic::DartState {
   static DartState* From(Dart_Isolate isolate);
   static DartState* Current();
 
+  const ftl::RefPtr<ftl::TaskRunner>& ui_task_runner() {
```

Comment:
Yeah, I thought about moving where the static is defined so I could see it from image_decoders.cc.